### PR TITLE
centos security updates check

### PR DESCRIPTION
**Pull request by keladhruv**
This PR checks for missing security updates on a centos image.

### CODE REVIEW

File: packer/scripts/99-img_check.sh [hunk 1]
```diff
@@ -359,9 +359,13 @@ function checkUpdates {
     elif [[ $OS == "CentOS Linux" ]]; then
         echo -en "\nChecking for available updates with yum, this may take a minute...\n\n"
         
-        update_count=$(yum list updates -q | grep -vc "Updated Packages")
+        update_count=$(yum list updates --security -q | grep -vc "Updated Packages")
```
Comment (caquillo07) [inline packer/scripts/99-img_check.sh:362]: were you able to test this? maybe some sample output? just curious

Comment (keladhruv) [reply packer/scripts/99-img_check.sh:362]: My system has all the relevant security updates, that being said the sample output mentions if there are any security updates available. 
`yum list updates --security
Last metadata expiration check: 0:04:22 ago on Wed 10 Feb 2021 02:45:37 PM UTC.
No security updates needed, but 29 updates available`

**Review by caquillo07 — APPROVED** [review]
this looks ok to me, but someone in the team with more CentOS knowledge should see htis before it gets merged

**Comment by pachamaltese** [comment]
@keladhruv checking now...

**Comment by pachamaltese** [comment]
@keladhruv I'm 100% used to Ubuntu, so I created a CentOS droplet to test, and it looks ok to me
```
[root@centos-s-1vcpu-1gb-nyc1-01 ~]# yum list updates --security -q | grep -vc "Updated Packages"
0
```

**Pull request merged**